src/doc/man/cargo-rustc.md

```diff
@@ -64,6 +64,10 @@ The `rustc` subcommand will treat the following named profiles with special beha
 See the [the reference](../reference/profiles.html) for more details on profiles.
 {{/option}}
 
+{{#option "`--crate-type` _crate-type_" }}
+Build with the given crate types.
+{{/option}}
```

Comment:
We generally don't document unstable options in the documentation until it is stabilized.  Can you remove this from the man page, and add a new section to https://doc.rust-lang.org/cargo/reference/unstable.html that follows the form of the other features?  That is, provide a brief description what it does, links to the RFC and tracking issue, and a short example of how to use it.